scripts/build/rollup-plugin-javascript-prefixer.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable import/no-extraneous-dependencies */
+const glob = require('glob');
+const fs = require('fs');
+
+const componentPackageJson = require(`${process.cwd()}/package.json`);
+
+const jsFilesToPrefix = glob
+  .sync('**/*.js', {
+    ignore: [
+      'node_modules/**/*',
+      'dist/**/*',
+      'lib/**/*',
+      'index.react.js',
+      '**demo.js',
+      '**.bak',
+      '**test**',
+      '**story**',
+    ],
+  })
+  .map(f => `./${f}`);
+
+const _getComponentInfo = () => {
+  const elementMapping = new Map();
+  elementMapping.set('10-atoms', 'a-');
+  elementMapping.set('20-molecules', 'm-');
+  elementMapping.set('30-organisms', 'o-');
+
+  const cwdAsStringArray = process.cwd().split('/');
+  const customElementName = componentPackageJson.name.replace('@axa-ch/', '');
+  const customElementTypePrefix = elementMapping.get(
+    cwdAsStringArray[cwdAsStringArray.length - 2]
```

Comment:
`  const componentDirectoryName = cwdAsStringArray[cwdAsStringArray.length - 2];
`
This part of your code is now better readable. But i mean, why the number "2". Where is this number comming from? Set a comment why a 2 and not a 3 or 4 (do you know what i mean?!). Or better set a const, f.a. `const dasVorletzteElementimArray = 2;`

Reply:
I usually use variable names to declare what is the value contained. From the variable name you see that we extract the componentDir from the whole path. Do you think I should be more explicit? Because "2" is just the second last element in the array.